print/locales/en-CH-scout.json

```diff
@@ -1 +1,9 @@
-{}
+{
```

Comment:
This file should probably be empty. I think there was a mistake between source and destination.